PWGCF/FEMTOSCOPY/FemtoDream/AliFemtoDreamTrackHist.h

```diff
@@ -141,6 +141,16 @@ class AliFemtoDreamTrackHist {
 
 
   void FillDCAXYPtBins(float pT, float dcaxy, int multiplicity);
+
+  void FillTOFMass(float mom, float beta){
+      if (beta>=0) {
+      fTOFMass->Fill(mom, mom/beta*TMath::Sqrt(1-beta*beta));
+      }
+      else {
+      fTOFMass=0;
```

Comment:
Here you reset the histogram pointer, what you want to do is Fill 0, Error code (e.g. -999)